components/camel-cassandraql/pom.xml

```diff
@@ -45,8 +45,13 @@
 
         <!-- cassandra -->
         <dependency>
-            <groupId>com.datastax.cassandra</groupId>
-            <artifactId>cassandra-driver-core</artifactId>
+            <groupId>com.datastax.oss</groupId>
+            <artifactId>java-driver-core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.datastax.oss</groupId>
+            <artifactId>java-driver-query-builder</artifactId>
+            <version>4.7.2</version>
```

Comment:
This should go in parent pom

Reply:
sounds great, I'll rework it and I think that tests could be allowed on CI with testcontainers. (I'v kept cassandra-util as it was used before)

Reply:
I'll fix that version, I've missed it